File: AdminPanel/client/src/actions/episode.js
import axios from "axios";

export const addEpisode = async (id,title, serialId, duration, number, videoLink, previewVideo) => {
    try {
        const response = await axios.post(
            "http://localhost:5000/episodes/addEpisode",
            {
                id,
                title,
                serialId,
                duration,
                number,
                videoLink,
                previewVideo
            }
        );
    } catch (e) {
        alert(e);
    }
}

export const deleteEpisode = async (id) => {
    try {
        const response = await axios.post(
            "http://localhost:5000/episodes/deleteEpisode",
            {
                id
            }
        );
    } catch (e) {

    }
}

export const updateEpisode = async (id, title, serialId, duration, number, videoLink, previewVideo) => {
    try {
        const response = await axios.post(
            "http://localhost:5000/episodes/updateEpisode",
            {
                id,
                title,
                serialId,
                duration,
                number,
                videoLink,
                previewVideo
            }
        );
    } catch (e) {
        alert(e);
    }
}

export const episodesBySerialId = async (serialId) => {
    try {
        const response = await axios.get(
            `http://localhost:5000/episodes/episodesBySerialId?serialId=${serialId}`,
        );
    } catch (e) {
        alert(e);
    }
}

File: NetFlexAPI/TestProject1/Models/Episode.cs
using System;
using Ply;

namespace TestProject1;

public class Episode
{
    public Guid Id { get; set; }
    public string Title { get; set; }
    public Guid SerialId { get; set; }
    public int Duration { get; set; }
    public int Number { get; set; }
    public string VideoLink { get; set; }
    public string PreviewVideo { get; set; }

    public Episode(Guid id,string title,Guid serialId, int duration, int number,string videoLink, string previewVideo)
    {
        Id = id;
        Title = title;
        SerialId = serialId;
        Duration = duration;
        Number = number;
        VideoLink = videoLink;
        PreviewVideo = previewVideo;
    }
}